buildspec.yml

```diff
@@ -0,0 +1,12 @@
+version: 0.2
+
+phases:
+  build:
+    commands:
+      - make codebuild 
+artifacts:
+  files:
+      - 'out/**/*'
+      - 'misc/windows-deploy/hostsetup.ps1'
```

Comment:
I think instead of doing `agent.exe` and `hostsetup.ps1` like this, can you just use the [zip file](https://github.com/aws/amazon-ecs-agent/blob/v1.16.1/scripts/stage#L74-L91) (the logic will probably need to be extracted from there, and then you can do it in `post_build`)?